src/components/hooks/use-livechat.js

```diff
@@ -42,6 +42,10 @@ export const useLivechat = () => {
                         setLiveChatInteractive(true)
                         if (is_livechat_query?.toLowerCase() === 'true') {
                             window.LC_API.open_chat_window()
```

Comment:
```suggestion
                            await window.LC_API.open_chat_window()
```
can we try awaiting this instead of setting a timeout?

Reply:
Either that or we use an `on open` event to adjust the weight 

```
LC_API.on_chat_window_opened = function() {
  document.getElementById('chat-widget-container').style.width = '450px'
};
```